lib/presentation/home/chats/chat_controller.dart

```diff
@@ -1,24 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_firebase_test/data/models/create_chat.dart';
 import 'package:flutter_firebase_test/data/repositories/chat.dart';
 import 'package:get/get.dart';
 import 'package:get_it/get_it.dart';
 
 class ChatController extends GetxController {
   final ChatRepository chatRepository = GetIt.instance.get<ChatRepository>();
 
+  // @override
+  // void onInit() {
+  //   // TODO: implement onInit
+
+  //   super.onInit();
+  // }
+
   final _state = ChatState().obs;
   ChatState get state => _state.value;
 
   RxBool isLoading = false.obs;
   RxString errorMessage = ''.obs;
 
-  Future<void> createChat(int consultantId) async {
+  Future<CreateChat> createChat(
```

Comment:
@Sparklinglily Your create chat function returns an error. I think the error is from your code, because the endpoint actually works